portal-web/test/functional/com/liferay/portalweb/macros/clay/ClaySamplePortlet.macro

```diff
@@ -1,11 +1,34 @@
 definition {
 
 	macro assertNavTabVisualContent {
+		var tabName = StringUtil.toLowerCase("${tab}");
+
 		Navigator.gotoNavTab(navTab = "${tab}");
 
-		OcularAssertElementImage(
-			locator1 = "ClaySamplePortlet#CLAY_SECTION",
-			value1 = "ClaySmoke/CanRenderUI/${tab}.png");
+		if ("${tab}" == "Navigation Bars") {
+			OcularAssertElementImage(
+				locator1 = "ClaySamplePortlet#CLAY_SECTION",
+				value1 = "clay_smoke/can_render_ui/navigation_bars.png",
+				value2 = "70");
+		}
+		else if ("${tab}" == "Pagination Bars") {
+			OcularAssertElementImage(
+				locator1 = "ClaySamplePortlet#CLAY_SECTION",
+				value1 = "clay_smoke/can_render_ui/pagination_bars.png",
+				value2 = "70");
+		}
+		else if ("${tab}" == "Progress Bars") {
+			OcularAssertElementImage(
+				locator1 = "ClaySamplePortlet#CLAY_SECTION",
+				value1 = "clay_smoke/can_render_ui/progress_bars.png",
+				value2 = "80");
+		}
```

Comment:
You can probably just use [StrintUtil.replace](https://github.com/liferay/liferay-qa-ee/blob/liferay-qa-docs/poshi-docs/references/pages/Methods.markdown#stringutil) to replace spaces with underscore so you can remove the if else statements. Please see link for other references to poshi methods, I know these aren't very public or easy to find...and the docs aren't maintained.

Implementation of more current methods can be found at: https://github.com/liferay/liferay-portal/blob/master/modules/test/poshi/poshi-core/src/main/java/com/liferay/poshi/core/util/StringUtil.java#L681-L701